### PR TITLE
Add chain 61

### PR DESCRIPTION
**Pull request by ghost**
# Add "New" Chain 61 (Ethereum Classic) and NEW RPC DUE OFFICIAL PRESS RELEASE!

Hey,

## Ethereum Classic here

Lets add this beauty to net-list! New RPC set ([Here the alert about rpc from official](https://ethereumclassic.org/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023))

Tests done

<sub>View in Huly <a href="https://app.huly.io/guest/w-kaanuzdogan-sourcify-65d30de0-75121c5c6c-fd615e?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NWU1ODIzMThiNmVjZDA0ZDYyZDkwYmMiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6Incta2FhbnV6ZG9nYW4tc291cmNpZnktNjVkMzBkZTAtNzUxMjFjNWM2Yy1mZDYxNWUiLCJwcm9kdWN0SWQiOiIifQ.3osPhwzyixe3RNRCs3wEBz0c-HJkbJexTgTCPwAr9Yg">HI-658</a></sub>

### CODE REVIEW

**Comment by ksandr84on** [comment]
@kuzdogan 
Hey, all done due instructions and test-new-chain passing. Only one thing - I'm added new rpc. The official link about  rpc change - https://ethereumclassic.org/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023

**Comment by ksandr84on** [comment]
Anybody home?

**Comment by marcocastignoli** [comment]
it would be the best to update the rpc directly in https://github.com/ethereum-lists/chains

**Comment by ksandr84on** [comment]
new rpcUrl on https://github.com/ethereum-lists/chains UPDATED. 

So guys, why there is NO ethereum classic in https://sourcify.dev/ ui? Choose chain - > No Ethereum classic

@marcocastignoli 
@kuzdogan

File: src/sourcify-chains.ts [hunk 1]
```diff
@@ -198,6 +198,13 @@ const sourcifyChainsExtensions: SourcifyChainsExtensionsObject = {
     monitored: false,
     contractFetchAddress: generateEtherscanCreatorTxAPI("56"),
   },
+  "61": {
+    supported: true,
+    monitored: true,
```
Comment (kuzdogan) [inline src/sourcify-chains.ts:203]:
```suggestion
    monitored: false,
```

**Review by kuzdogan — CHANGES_REQUESTED** [review]
Please turn off the monitor as stated in the instructions. 

Otherwise looks good 👍

**Comment by ksandr84on** [comment]
@kuzdogan  done

**Pull request merged**